dynamic-plugins/imports/package.json

```diff
@@ -16,13 +16,14 @@
     "tar": "6.2.0"
   },
   "peerDependencies": {
-    "@janus-idp/backstage-plugin-aap-backend": "1.4.3",
+    "@janus-idp/backstage-plugin-3scale-backend": "1.3.4",
```

Comment:
This plugin is missing configuration in `app-config.dynamic-plugins.yaml` and `dynamic-plugins.default.yaml`